lisdf/plan_executor/robots/panda.py

```diff
@@ -18,7 +18,7 @@ def joint_configuration(self) -> np.ndarray:
         return self.configuration[:7]
 
     def set_joint_configuration(self, joint_configuration: np.ndarray) -> None:
-        self.configuration = np.concatenate(
```

Comment:
numpy should have type stubs already so mypy shouldn't be complaining here. Are you seeing the errors locally or on github?

Same for other type ignores below